src/clone-element.js

```diff
@@ -19,14 +19,11 @@ export function cloneElement(vnode, props, children) {
 		else normalizedProps[i] = props[i];
 	}
 
-	if (arguments.length > 3) {
-		children = [children];
-		for (i = 3; i < arguments.length; i++) {
-			children.push(arguments[i]);
-		}
-	}
 	if (arguments.length > 2) {
-		normalizedProps.children = children;
+		normalizedProps.children =
+			arguments.length > 3
+				? Array.prototype.slice.call(arguments, 2)
```

Comment:
Hoisting the .slice access out of this function should improve perf a tiny bit more:

```js
const slice = Array.prototype.slice;
// etc

		normalizedProps.children =
			arguments.length > 3 ? slice.call(arguments, 2) : children;
```

Reply:
Jep, and perhaps move it to util so it can be shared (less bytes)?

Reply:
Usually when something is only used in 2 places we duplicate - gzip compression makes the duplicates smaller than an intermediary variable.

Reply:
100% agreed on the size thing. It actually does still run for fork-PRs, but it can't post a comment. I need to use the workaround Andre came up with that lets benchmarks run.